docs/user/alerting/rule-types/geo-rule-types.asciidoc

```diff
@@ -10,24 +10,90 @@
-than the current time minus the amount of the interval. If data older than
-`now - <current interval>` is ingested, it won't trigger a rule.
+Entity locations are queried to determine whether they are contained within any monitored boundaries.
+Entity data should be somewhat "real time", meaning the dates of new documents aren't older than the current time minus the amount of the interval.
+If data older than `now - <current interval>` is ingested, it won't trigger a rule.
 
 [float]
-=== Actions
+=== Add actions
 
-A rule can be triggered either when a containment condition is met or when an entity is no longer contained.
+You can optionally send notifications when the rule conditions are met.
+In particular, this rule type supports:
+
+- alert summaries
+- actions that run when the containment condition is met
+- actions that run when an entity is no longer contained
+
+For each action, you must choose a connector, which provides connection information for a {kib} service or third party integration.
+For more information about all the supported connectors, go to <<action-types>>.
+
+After you select a connector, you must set the action frequency.
+You can choose to create a summary of alerts on each check interval or on a custom interval.
+Alternatively, you can set the action frequency such that actions run for each alert.
+Choose how often the action runs (at each check interval, only when the alert status changes, or at a custom action interval).
+You must also choose an action group, which indicates whether the action runs when the containment condition is met or when an entity is no longer contained.
+Each connector supports a specific set of actions for each action group.
+For example:
 
 [role="screenshot"]
 image::user/alerting/images/alert-types-tracking-containment-action-options.png[Action frequency options for an action,width=75%]
 // NOTE: This is an autogenerated screenshot. Do not edit it directly.
+
+You can further refine the conditions under which actions run by specifying that actions only run they match a KQL query or when an alert occurs within a specific time frame.
+
+[float]
+=== Add action variables
+
+You can pass rule values to an action to provide contextual details.
+To view the list of variables available for each action, click the "add rule variable" button.
+For example:
+
+[role="screenshot"]
+image::images/alert-types-tracking-containment-rule-action-variables.png[Passing rule values to an action,width=75%]
+// NOTE: This is an autogenerated screenshot. Do not edit it directly.
+
+The following action variables are specific to the tracking containment rule.
+You can also specify <<rule-action-variables,variables common to all rules>>.
+
+`context.containingBoundaryId`::
+The identifier for the boundary containing the entity.
+
+`context.containingBoundaryName`::
+The name of the boundary containing the entity.
+
+`context.detectionDateTime`::
+The alert interval end time this change was recorded.
```

Comment:
How about "The alert interval end time triggering the alert"?

Reply:
Another option:

> The end of the check interval( is this the same as the alert interval?) when the alert occurred.

NOTE: Whatever new value we use will need to be updated in the UI too: https://github.com/elastic/kibana/blob/main/x-pack/plugins/stack_alerts/server/rule_types/geo_containment/rule_type.ts

Reply:
Yes, the alert interval and check interval are the same. I like your suggestion of "The end of the check interval".

Reply:
Thanks! I made that change in 502eb92766aa4d698c33d231b5ad20cd612f5beb